### ISSUE TITLE
aliasMappings is parsing even the files that no file is requiring

### ISSUE THREAD

**Issue by fsuarezd** [jmreidy/grunt-browserify#76]
If on the aliasMappings example you add a file under the shared folder that no file is requiring and that requires a non existing file:

require("lib/fake")

then you won't be able to browserify the example.
I could make the example work again by doing:

aliasMappings: [
            {
              cwd: 'shared',
              src: ['**/models/testModel.js', '**/controllers/testController.js'],
              dest: 'lib',
            },
          ]

So basically it will be great if it wouldn't try to parse files that are not on the require tree.
Doest that makes sense?
Thanks
